StyleGuide.md

```diff
@@ -206,124 +322,142 @@ Try to address as many of the suggestions as possible. If Vale flags a word that
 
-### Tips, notes, and warnings
+- For a single placeholder value, use a follow-up sentence.
+- For multiple placeholder values, use a bulleted list.
+- For many placeholder values (10+), use a table.
+- For large code blocks, define the placeholder values inside the code block with an inline code comment.
 
-Our docs use three classes of highlighted text (also referred to as callouts):
+Ensure that placeholders are placed within backticks: `SET {session variable}`. This signifies that placeholder values are code.
 
-- [Tips](#tips)
-- [Notes](#notes)
-- [Warnings](#warnings)
+If the code sample is sensitive to curly bracket characters (as in JavaScript), you can use `<>` instead.
 
-The highlighting is generated using Liquid tags, each of which must be on its own line. You can use Markdown (preferred) or HTML within the highlighted text.
+### Callouts
 
-#### Tips
+CockroachDB docs use three classes of "callouts," which are highlighted blocks of text: tips, notes, and warnings.
 
-Use a tip to highlight nice-to-know pieces of information.
+- Use a _tip_ to highlight nice-to-know pieces of information.
 
-For example, you might include a tip to our GitHub repo's Terraform scripts on the Google Cloud Engine deployment page. It's nice to know it's there, but doesn't clarify anything nor is it critical.
+  For example, you might use a tip to link to our GitHub repo's Terraform scripts on the Google Cloud Engine deployment page. It's nice to know it's there, but doesn't clarify anything nor is it critical.
 
-To insert a tip, use the following code:
+- Use a _note_ to call attention to a piece of clarifying information. This information should **not** be crucial to accomplishing the task in the document.
 
-~~~
-{{site.data.alerts.callout_success}}
-<tip text goes here>
-{{site.data.alerts.end}}
-~~~
+  For example, you might use a note to let users know that the `DELETE` statement only deletes rows and that to delete columns you must use `ALTER TABLE`. This helps clarify `DELETE`'s purpose and point users to the right place.
 
-#### Notes
+- Use a _warning_ to express that a piece of information is critical to understand to prevent data loss, security vulnerabilities, or unexpected behavior.
 
-Use a note to call attention to a piece of clarifying information; this information should not be crucial to accomplishing the task in the document.
+  For example, you might include a warning that using `CASCADE` in `DROP INDEX` drops dependent objects without warning. This is critical to prevent users from unexpectedly losing constraints or additional indexes.
 
-For example, you might use a note to let users know that the `DELETE` statement only deletes rows and that to delete columns you must use `ALTER TABLE`. This helps clarify `DELETE`‘s purpose and point users to the right place.
+**Best practices:**
 
-To insert a note, use the following code:
+- Avoid placing callouts adjacent to each other.
```

Comment:
I would add "Avoid placing a callout immediately below a header without introductory text for the section" but IIRC we've done that in a couple places.